contributing/README.md

```diff
@@ -0,0 +1,28 @@
+# Contributing
+
+Contributing to the project does not necessarily require familiarity with every component. To find a quick breakdown
+of the smaller parts of this project, consult the [index](index.md), which breaks down the SDK Structure into small
+components and references to help contributors find the necessary references without having to go in-depth into the
+project's overall structure. For a more in-depth breakdown of the project structure, it would be beneficial for you
+to have a look at the [documentation](vmware.github.io/vmware-aria-operations-integration-sdk/),
+[issue tracker](https://github.com/vmware/vmware-aria-operations-integration-sdk/issues),
+[project structure](index.md), and basic functioning of every component.
```

Comment:
This is better, but now seems a bit wordy and redundant in places. It also links to the same page for a detailed breakdown as the overview page. Since we don't really have any more detailed information than what's linked from the index, I think contacting us is the best option. Maybe something like this?

```suggestion
Contributing to the project does not necessarily require familiarity with every component. The [index page](index.md) explains the project structure and links to some high-level project documentation. If you require detailed information not covered here, please open a [discussion](https://github.com/vmware/vmware-aria-operations-integration-sdk/discussions/new?category=general)
```